src/journal/bsod.c

```diff
@@ -34,19 +34,21 @@ static int help(void) {
         if (r < 0)
                 return log_oom();
 
-        printf("%s\n\n"
-               "%sFilter the journal to fetch the first message from the\n"
-               "current boot with an emergency log level and displays it\n"
-               "as a string and a QR code.\n\n%s"
+        printf("%1$s [OPTIONS...]\n\n"
+               "%5$sFilter the journal to fetch the first message from thecurrent boot with an%6$s\n"
```

Comment:
"the current"

There's something wrong with mode in this sentence: "Filter the journal … and displays it".